app/config/FeatureSwitches.scala

```diff
@@ -28,19 +28,19 @@ case class FeatureSwitches(featureSwitchConfig: Configuration) {
     val maybeVersion: Option[String] =
       version match {
         case versionRegex(v) => Some(v)
-        case _               => None
+        case _ => None
       }
 
     val enabled = for {
       validVersion <- maybeVersion
-      enabled      <- featureSwitchConfig.getOptional[Boolean](s"version-$validVersion.enabled")
+      enabled <- featureSwitchConfig.getOptional[Boolean](s"version-$validVersion.enabled")
     } yield enabled
 
     enabled.getOrElse(false)
   }
 
   val isTaxYearNotEndedRuleEnabled: Boolean = isEnabled("taxYearNotEndedRule.enabled")
-  val isTaxYearSpecificApiEnabled: Boolean  = isEnabled("tys-api.enabled")
+  val isTaxYearSpecificApiEnabled: Boolean = isEnabled("tys-api.enabled")
```

Comment:
I may be wrong but it looks like the scala formatting has been removed and that is the only change in this file? Can we format this again to be sure

Reply:
This is just how my scala formatter is working. I don't think I can change that.